src/datetime_queries/fetchers.rs

```diff
@@ -21,3 +21,24 @@ pub struct Fetchers {
     pub hour: FetchByHour,
     pub get_latest: GetLatestEntry,
 }
+impl Fetchers {
+    pub fn new(
+        day_to_day: FetchByDayDay,
+        day_to_hour: FetchByDayHour,
+        hour_to_day: FetchByHourDay,
+        hour_to_hour: FetchByHourHour,
+        day: FetchByDay,
+        hour: FetchByHour,
+        get_latest: GetLatestEntry,
+    ) -> Self {
+        Self {
+            day_to_day,
+            day_to_hour,
+            hour_to_day,
+            hour_to_hour,
+            day,
+            hour,
+            get_latest,
+        }
+    }
```

Comment:
I was thinking... set up one called `default` here too that constructs a new instance of all the parts. 

Then you can call it once in your tests, `let mut fetchers = Fetchers::default()` and then set up your expect_x against nested items on `fetchers`

Reply:
how would we indicate to make the instances mock or regular?